ui/vss-extension.json

```diff
@@ -139,6 +139,36 @@
                 "order": 110,
                 "uri": "workItemService.html"
             }
+        },
+        {
+            "id": "sample-work-item-menu",
+            "type": "ms.vss-web.action",
+            "description": "Shows the target properties for menu actions for work items",
+            "targets": [
+                "ms.vss-work-web.work-item-context-menu",
+                "ms.vss-work-web.active-work-item-notifications"
+            ],
+            "properties": {
+                "text": "Try me!",
+                "title": "ms.vss-work-web.work-item-context-menu",
```

Comment:
Does the title have to be this? Or can it be any text? Where does it even show up? Is it required?